scholia/app/templates/work_data.sparql

```diff
@@ -41,6 +41,15 @@ WHERE {
     GROUP BY ?dummy
   }
   UNION
+  {
+    BIND(2.5 AS ?order)
+    BIND("Language" AS ?description)
+    ?work wdt:P407 ?iri .
+    ?iri rdfs:label ?value_string .
+    FILTER (LANG(?value_string) = 'en')
```

Comment:
This will only show for English works?

Reply:
oh nevermind, that the label itself is in English

Reply:
oh, but a good call, nevertheless! this should be taken from the active UI... hang on, will fix this!

Reply:
Actually, the full query has this... I'll file an issue for now to check all `FILTER` in all SPARQL instead, if you're okay.

Reply:
Filed issue: https://github.com/WDscholia/scholia/issues/1640